junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestExtension.java

```diff
@@ -39,6 +39,7 @@
 
 	private static final String METHOD_CONTEXT_KEY = "context";
 	static final String ARGUMENT_MAX_LENGTH_KEY = "junit.jupiter.params.displayname.argument.maxlength";
+	static final String DISPLAY_NAME_TEMPLATE_KEY = "junit.jupiter.params.displayname.template";
```

Comment:
I know I said "template" in the other issue. Looking at the sources of `@ParameterizedTest` now I think "pattern" would be better. Looking at other configuration paramters, though, I think `junit.jupiter.params.displayname.default` would be more consistent.

@junit-team/junit-lambda WDYT?

Reply:
The term "pattern" is consistent with the existing documentation for `DEFAULT_DISPLAY_NAME`:

> Default display name pattern for the current invocation of a `@ParameterizedTest` method: `"[{index}] {argumentsWithNames}"`

Reply:
> Looking at other configuration paramters, though, I think `junit.jupiter.params.displayname.default` would be more consistent.

Yes, let's go with `junit.jupiter.params.displayname.default`.